src/polkitbackend/polkitbackendauthority.c

```diff
@@ -1591,3 +1598,24 @@ polkit_backend_authority_log (PolkitBackendAuthority *authority,
 
   g_free (message);
 }
+
+void
+polkit_backend_authority_set_log_level (const gchar *level)
+  {
+    if (g_strcmp0 (level, "debug") == 0)
```

Comment:
This doesn't allow using numerical values for shortopt. Using sth like "polkitd -l crit" doesn't look like a standard solution.

Reply:
I have never seen any project expose this with numeric values? In systemd we do use the string names though, see SYSTEMD_LOG_LEVEL and LogControl1

https://www.freedesktop.org/software/systemd/man/latest/systemd.html#%24SYSTEMD_LOG_LEVEL
https://www.freedesktop.org/software/systemd/man/latest/org.freedesktop.LogControl1.html

I don't think it's necessary to expose this option as a numeric value for users, it's something that programmers might understand, but it would be very opaque for a normal user

Reply:
Sorry, but using numerical values would be confusing (for administrators, mind you), but using sometimes shortened words for shortopts wouldn't?
This is not an intention to massage my ego, but it simply doesn't make sense or consistency.
I've seen many standard programs that use shortopt with a number and longopt with a string.
If polkitd wasn't such a small program expected to be used frequently from shell directly, I'd be for creating one GOption "-l" with nums and another one for "--log-levels" for textual parameters, which would be consistently uncut.

Reply:
I meant for log levels. I don't remember anything common enough that takes parameters for the log level as an opaque number. The enum is an implementation detail of syslog after all, the way it's exposed to users is via these strings